app/meldingen/incident/containers/IncidentContainer/saga.js

```diff
@@ -1,6 +1,49 @@
-// import { take, call, put, select } from 'redux-saga/effects';
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+import request from 'utils/request';
+
+import { CREATE_INCIDENT, GET_CLASSIFICATION } from './constants';
+import {
+  createIncidentSuccess,
+  createIncidentError,
+  getClassificationSuccess,
+  getClassificationError
+} from './actions';
+import makeSelectIncidentContainer from './selectors';
+
+export function* createIncident() {
+  const requestURL = '/api/meldingen';
+
+  try {
+    const data = yield select(makeSelectIncidentContainer());
+    console.log('data', data);
+    const incident = yield call(request, requestURL);
+    yield put(createIncidentSuccess(incident));
+  } catch (err) {
+    yield put(createIncidentError(err));
+  }
+}
+
+export function* getClassification({ text }) {
+  const requestURL = 'http://meldingen-classification.herokuapp.com/calls/';
+
+  try {
+    const classification = yield call(request, requestURL, {
+      method: 'post',
+      body: JSON.stringify({
+        text
+      }),
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    });
+    yield put(getClassificationSuccess(classification));
+  } catch (err) {
+    yield put(getClassificationError(err));
+  }
+}
 
 // Individual exports for testing
-export default function* defaultSaga() {
-  // See example in containers/HomePage/saga.js
+export default function* watchIncidentContainerSaga() {
+  yield takeLatest(CREATE_INCIDENT, createIncident);
+  yield takeLatest(GET_CLASSIFICATION, getClassification);
```

Comment:
the two yields are waiting for each other. use 
yield [
takeLatest(...),
takeLatest(...)
]

to start them in parallel